src/util.ts

```diff
@@ -563,6 +562,28 @@ export function checkDirectoryExists(filePath: string): Promise<boolean> {
   });
 }
 
+// Read the files in a directory.
+export function readDir(dirPath: string): Promise<string[]> {
+  return new Promise((resolve) => {
+    fs.readdir(dirPath, (_err, list) => {
```

Comment:
In the case of `_err`, is the list an empty array?  Maybe it's best to check for `_err` and force return `[]`.